Signal/src/ViewControllers/ConversationView/MessagesViewController.m

```diff
@@ -488,6 +488,9 @@ - (void)configureForThread:(TSThread *)thread
 
     [self.uiDatabaseConnection beginLongLivedReadTransaction];
 
+    self.page = 0;
+    [self ensureDynamicInteractions];
```

Comment:
We need to update the "unread indicator" _before_ we determine the initial range size, since it depends on where the unread indicator is placed.